app/models/category.rb

```diff
@@ -0,0 +1,8 @@
+class Category < ApplicationRecord
+
+  has_many :users
+
+  validates :name, presence: true
+  
+  validates :name, uniqueness: true
```

Comment:
@meganft I am actually not super clear on how this is set up, but I think I agree with you. @bfpepper can you walk us through your thought process on this user setup? On Slack if that's easier. Thanks!